packages/ui/src/theme/css/component/fieldGroup.scss

```diff
@@ -1,77 +1,168 @@
 %remove-start-radii {
-  border-start-start-radius: 0;
-  border-end-start-radius: 0;
+  border-start-start-radius: 0; // border-top-left-radius (ltr reading mode)
+  border-end-start-radius: 0; // border-bottom-left-radius (ltr reading mode)
+
+  // required for Safari 14 and below
+  // see: https://caniuse.com/mdn-css_properties_border-end-end-radius
+  @supports not (border-start-start-radius: 0) {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+  }
 }
 %remove-end-radii {
-  border-end-end-radius: 0;
-  border-start-end-radius: 0;
+  border-end-end-radius: 0; // border-bottom-right-radius (ltr reading mode)
+  border-start-end-radius: 0; // border-top-right-radius (ltr reading mode)
+
+  // required for Safari 14 and below
+  // see: https://caniuse.com/mdn-css_properties_border-end-end-radius
+  @supports not (border-end-end-radius: 0) {
+    border-bottom-right-radius: 0;
+    border-top-right-radius: 0;
+  }
 }
 
 .amplify-field-group {
   gap: var(--amplify-components-fieldgroup-gap);
 
   // Input styling
-  .amplify-input:not(:last-child) {
+  :not(:last-child) .amplify-input {
     @extend %remove-end-radii;
   }
-  .amplify-input:not(:first-child) {
+  :not(:first-child) .amplify-input {
     @extend %remove-start-radii;
   }
+}
+.amplify-field-group__outer-start,
+.amplify-field-group__outer-end {
+  display: flex;
+  .amplify-field-group__control {
+    height: 100%;
+  }
+}
 
-  .amplify-field-group__start,
-  .amplify-field-group__end {
-    display: flex;
-    .amplify-field-group__control {
-      height: 100%;
+/**
+ * Outer field group components
+ */
+.amplify-field-group__outer-start {
+  .amplify-field-group__control {
+    @extend %remove-end-radii;
+
+    // This targets non first-child controls
+    // when there are multiple controls
+    &:not(:first-child) {
+      @extend %remove-start-radii;
+      @extend %remove-end-radii;
     }
   }
-
-  .amplify-field-group__start {
-    .amplify-field-group__control {
+  // Select requires special rules due to icon
+  .amplify-select__wrapper {
+    .amplify-select {
       @extend %remove-end-radii;
-
-      // This targets non first-child controls
-      // when there are multiple controls
-      &:not(:first-child) {
-        @extend %remove-start-radii;
-        @extend %remove-end-radii;
-      }
     }
-    // Select requires special rules due to icon
-    .amplify-select__wrapper {
+    &:not(:first-child) {
       .amplify-select {
+        @extend %remove-start-radii;
         @extend %remove-end-radii;
       }
-      &:not(:first-child) {
-        .amplify-select {
-          @extend %remove-start-radii;
-          @extend %remove-end-radii;
-        }
-      }
     }
   }
-  .amplify-field-group__end {
-    .amplify-field-group__control:not(.amplify-select) {
-      @extend %remove-start-radii;
+}
+.amplify-field-group__outer-end {
+  .amplify-field-group__control:not(.amplify-select) {
```

Comment:
Is `:not(.amplify-select)` required here? I think the following rules for Select should override it because I don't see this used in `.amplify-field-group__outer-start`